src/handlers/commands.js

```diff
@@ -67,21 +64,19 @@ module.exports.setupSlashCommands = async client => {
 
 const respondWithError = ({ id, token }, content) => client.api.interactions(id, token).callback.post({ data: { type: 3, data: { flags: 64, content }}})
 
-module.exports.registerSlashCommands = async () => {
-  await commandLoader;
-
+module.exports.registerSlashCommands = async client => {
   // remove old commands
-  const slashCommands = await client.api.applications(config.id).commands.get();
+  const slashCommands = await client.api.applications(client.user.id).commands.get();
   await Promise.all(slashCommands
     .filter(c => !commands.get(slashCommand.name))
```

Comment:
`slashCommand` is not defined

Reply:
Fixed in https://github.com/countr/countr/pull/83/commits/c6b4fb30f053d84a94eed6278523f4ccb4e86b43